test/config/utility.cc

```diff
@@ -113,6 +113,13 @@ std::string ConfigHelper::tcpProxyConfig() {
 )EOF");
 }
 
+std::string ConfigHelper::tlsInspectorFilter() {
```

Comment:
cc @lizan who I think added something similar in a current PR and will need a merge.

Reply:
This one? https://github.com/envoyproxy/envoy/pull/10569
This PR is on top of the above